metrotranscom/doorway
# feat: add basic Google Map

This adds basic skeleton functionality of a Google Map (no UX work yet). Right now its hardcoded to send you to the listing page when you click on a pin.

Pins are concentrated in 3 places because our seed data is built like this -- we should update seed data to be more geographically interesting but that can be done in a followup.

Steps to test:
1. update env file with a Google Maps API key (unfortunately I cannot provide one)
2. Open /listings and observe map, also try to click on pins.

![image](https://user-images.githubusercontent.com/10789523/233472739-f6f1be26-3d06-424f-8e6e-260ade634b4b.png)
